client/src/store/modules/technologies/index.ts

```diff
@@ -0,0 +1,19 @@
+import {actions} from './actions';
+import {mutations} from './mutations';
+import {ITechnology} from '../../../shared/interfaces/ITechnology';
+
+export interface ITechnologyState {
+  technologies: ITechnology[];
+  loading: boolean;
+}
+
+const initialState: ITechnologyState = {
+  technologies: [],
+  loading: true
+};
+
+export default {
+  state: {...initialState},
```

Comment:
Just do it inline, no need for additional variable, annotate the whole state Module <ITechnologyState, {}>